docs/miner/email_notification_guide.md

```diff
@@ -0,0 +1,185 @@
+# Email Notification Guide for Proof Submission
+
+This guide describes how to configure and use the email notification feature in EthStorage's mining process. This feature is useful for storage providers or es-node operators who want to receive timely notifications about proof submissions and their results.
+
+---
+
+## How It Works
+
+If enabled, storage providers will receive email notifications for the following mining events:
+
+1. When a shard finishes syncing data via P2P and begins its mining task. This happens when a node is started for the first time requiring its local storage to catch up with the network.
+2. When a storage proof is generated and submitted to the smart contract as a mining transaction.
+
+## SMTP Service Preparation
+
+**You must have access to a working SMTP service to use the email notification feature.**  
+If you do not already have an SMTP account, you need to apply for one yourself. Below are two common options:
+
+### Gmail
+
+- You can use your Gmail account as an SMTP sender.
+- SMTP Host: `smtp.gmail.com`
+- SMTP Port: `587`
+- Reference: [Send email from a printer, scanner, or app](https://support.google.com/a/answer/176600?hl=en) (Option 2).
+
+### Mailjet
+
+- [Mailjet](https://app.mailjet.com/signup) is a third-party email service provider that offers free and paid SMTP services.
+- SMTP Host: `in-v3.mailjet.com`
+- SMTP Port: `587`
+- Reference: [How can I configure my SMTP parameters?](https://documentation.mailjet.com/hc/en-us/articles/360043229473-How-can-I-configure-my-SMTP-parameters)
+
+For other providers, refer to their documentation for SMTP host, port, and credential setup.
+
+---
+
+
+## Email Configuration 
+
+You can configure email notifications via CLI flags or environment variables. The most common way is to pass parameters to your node startup scripts where storage mining is enabled (e.g., `run.sh` or `run-l2.sh`):
+
+```bash
+  ...
+  --miner.email-enabled \
```

Comment:
It would be helpful to provide users with a way to quickly verify their email configuration by sending a test email immediately

Reply:
It can now be done with the `es-node email` command. Guide updated accordingly.